### PR TITLE
Travis should fail when failures happen

### PR DESCRIPTION
**Pull request by willbarton**
Our Travis configuration wasn't failing the build when tests failed. This PR fixes that and fixes the test failures that got introduced in the meantime.

## Checklist

* [ ] PR has an informative and human-readable title
* [ ] Changes are limited to a single goal (no scope creep)
* [ ] Code can be automatically merged (no conflicts)
* [ ] Code follows the standards laid out in the [development playbook](https://github.com/cfpb/development)
* [ ] Passes all existing automated tests
* [ ] Any *change* in functionality is tested
* [ ] New functions are documented (with a description, list of inputs, and expected output)
* [ ] Placeholder code is flagged
* [ ] Visually tested in supported browsers and devices
* [ ] Project documentation has been updated
* [ ] Reviewers requested with the [Reviewer tool](https://help.github.com/articles/about-pull-request-reviews/) :arrow_right:

### CODE REVIEW

File: cfgov/unprocessed/js/routes/on-demand/feedback-form.js [hunk 1]
```diff
@@ -3,8 +3,8 @@
    ========================================================================== */
 
 
-const COMMENT_ERRORS = require( '../../config/error-messages-config' ).COMMENT || {};
-const OPTION_ERRORS = require( '../../config/error-messages-config' ).OPTION || {};
+const COMMENT_ERRORS = require( '../../config/error-messages-config' ).COMMENT;
```
Comment (anselmbradford) [inline cfgov/unprocessed/js/routes/on-demand/feedback-form.js]: Was there a scenario where this fallback was needed? Should this check the value of `COMMENT_ERRORS` further down and set it to an empty object if it is undefined?

File: test/browser_tests/cucumber/step_definitions/multi-select.js [hunk 2]
```diff
@@ -15,12 +15,12 @@ defineSupportCode( function( { Then, When, Before } ) {
...
-      await browser.wait( EC.visibilityOf( multiSelect.elements.search ) );
-      await multiSelect.elements.search[searchInputAction]();
-    }
-  );
+  // When( /I (.*) on the multi-select search input/,
+  //   async function( searchInputAction ) {
+  //     await browser.wait( EC.visibilityOf( multiSelect.elements.search ) );
+  //     await multiSelect.elements.search[searchInputAction]();
+  //   }
+  // );
```
Comment (anselmbradford) [inline test/browser_tests/cucumber/step_definitions/multi-select.js]: Run `gulp lint --fix` it'll want this to be `/* */` comment.

File: cfgov/unprocessed/js/routes/on-demand/feedback-form.js (hunk 1 above)
Comment (sebworks) [reply cfgov/unprocessed/js/routes/on-demand/feedback-form.js]: It's a config file; why would it not return an object?

Comment (anselmbradford) [reply cfgov/unprocessed/js/routes/on-demand/feedback-form.js]: Yeah it doesn't look needed to me, but it was also added. Just not sure if there was a reason.

File: test/browser_tests/cucumber/step_definitions/multi-select.js (hunk 2 above)
Comment (sebworks) [reply test/browser_tests/cucumber/step_definitions/multi-select.js]: fixed in https://github.com/cfpb/cfgov-refresh/pull/3706/commits/349995d72e24ecc6af9cc61915ef7ca667b0e1e4

**Pull request merged**